### PR TITLE
Feature/115 unit tests shortucts logo

### LINKED ISSUE #115: Unit tests for shortcuts and logo components

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #115

### CODE REVIEW

File: src/app/common/state-handler/state-handler.service.spec.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+
 describe('State handler service', () => {
   let $ngRedux;
   let stateHandlerService;
 
   beforeEach(setup);
 
   it('should dispatch an action', () => {
-    stateHandlerService.dispatch('disableBusinessPartnerMenu', {});
+    stateHandlerService.dispatch(VisibilityActionsConstants.DISABLE_BUSINESS_PARTNER_MENU_ACTION, {});
```
Comment (frontendherodk) [inline src/app/common/state-handler/state-handler.service.spec.js]: Wouldn't it be enough if we just called it `VisibilityActions` ?

File: src/app/common/visibility/visibility.actions.js [hunk 2]
```diff
@@ -1,77 +1,77 @@
-import * as constants from './visibility.constants';
+import * as constants from './visibility.actions.constants';
 
 const VisibilityActions = () => {
 
   const showApplicationsMenu = () => {
     return {
-      type: constants.SHOW_APPLICATIONS_MENU
+      type: constants.SHOW_APPLICATIONS_MENU_TYPE
```
Comment (frontendherodk) [inline src/app/common/visibility/visibility.actions.js]: And here call it `actions.SHOW_APPLICATIONS_MENU`

File: src/app/common/visibility/visibility.actions.constants.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+// ACTIONS
+export const SHOW_APPLICATIONS_MENU_ACTION = 'showApplicationsMenu';
```
Comment (frontendherodk) [inline src/app/common/visibility/visibility.actions.constants.js]: Why are the actions and types different?

Comment (dami-gg) [inline src/app/common/visibility/visibility.actions.constants.js]: after discussion, I'll change them and merge them into one type. It makes sense, but I'll do it in the other PR and merge it back here

File: src/app/common/state-handler/state-handler.service.spec.js (hunk 1 above)
Comment (dami-gg) [inline src/app/common/state-handler/state-handler.service.spec.js]: one thing is the actions and one the constants which map the dispatch with the action to trigger, I'd keep like it is in this case

**Comment by dami-gg** [comment]
I've just updated this PR with the requested changes which I performed in the other PR and are now merged to master @frontendherodk

**Comment by frontendherodk** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**